components/Searchbar/styles.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components/native";
+
+// export const Container = styled.View`
+//     position: absolute;
+//     top: 0;
+//     left: 0;
+//     width: 200px;
+//     bottom: 0;
+//     background-color: red;
+// `
+
+export const Container = styled.View`
```

Comment:
Don't position this absolutely yourself! There is a `Content` component in `Home` that exists precisely for the purpose of absolutely-positioning these overlay elements over the map view.